README-VIM.md

```diff
@@ -159,6 +159,47 @@ let g:fzf_colors =
 let g:fzf_history_dir = '~/.local/share/fzf-history'
 ```
 
+The above `g:fzf_colors` setting is a little tricky.
```

Comment:
I'd say this is unnecessary to add, let people judge how tricky it is on their own :)